lib/routes/user/home/home_page.dart

```diff
@@ -134,6 +135,7 @@ class HomeState extends State<Home> {
         child: new Scaffold(
             key: _scaffoldKey,
             appBar: new AppBar(
+              brightness: theme.themeId == "BLUE" ? Brightness.light : Theme.of(context).appBarTheme.brightness,
```

Comment:
Why instead not assigning `Brightness.light` to the dark theme?
Also according to documentation we don't need to `assign AppBar.brightness` explicitly as it is taken from the theme automatically: https://api.flutter.dev/flutter/material/AppBar/brightness.html

Reply:
This is a special case, on blue theme the wallet dashboard's background is white and status bar text & icon would not be visible with brightness defined. in app theme.